handlers/register.go

```diff
@@ -71,6 +71,11 @@ func Register(mainRouter *mux.Router, cfg *config.Config, kSvc services.KafkaSer
 	appRouter.HandleFunc("/delta/filing-history", NewDeltaHandler(kSvc, h, chv, cfg, false, false, cfg.FilingHistoryDeltaTopic).ServeHTTP).Methods(http.MethodPost).Name("filing-history-delta")
 	appRouter.HandleFunc("/delta/filing-history/validate", NewDeltaHandler(kSvc, h, chv, cfg, true, false, cfg.FilingHistoryDeltaTopic).ServeHTTP).Methods(http.MethodPost).Name("filing-history-delta-validate")
 	appRouter.HandleFunc("/delta/filing-history/delete", NewDeltaHandler(kSvc, h, chv, cfg, false, true, cfg.FilingHistoryDeltaTopic).ServeHTTP).Methods(http.MethodPost).Name("filing-history-delete-delta")
+	appRouter.HandleFunc("/delta/document-store", NewDeltaHandler(kSvc, h, chv, cfg, false, false, cfg.DocumentStoreDeltaTopic).ServeHTTP).Methods(http.MethodPost).Name("document-store-delta")
+	appRouter.HandleFunc("/delta/document-store/validate", NewDeltaHandler(kSvc, h, chv, cfg, true, false, cfg.DocumentStoreDeltaTopic).ServeHTTP).Methods(http.MethodPost).Name("document-store-delta-validate")
+	appRouter.HandleFunc("/delta/registers", NewDeltaHandler(kSvc, h, chv, cfg, false, false, cfg.RegistersDeltaTopic).ServeHTTP).Methods(http.MethodPost).Name("registers-delta")
+	appRouter.HandleFunc("/delta/registers/delete", NewDeltaHandler(kSvc, h, chv, cfg, true, false, cfg.RegistersDeltaTopic).ServeHTTP).Methods(http.MethodPost).Name("registers-delta-delete")
```

Comment:
boolean params should be swapped for /delta/registers/delete, first boolean is doValidationOnly and second boolean is isDelete